oidc_client/config/config.ini

```diff
@@ -78,6 +78,9 @@ url_callback=localhost:8080/callback
 # URL the OIDC Client should redirect to after authentication
 url_redirect=localhost:5000
 
+# URL to the token revocation endpoint at AAI
+url_revoke=https://login.elixir-czech.org/oidc/revoke
+
 # Claims requested for access token, for multiple values separate scopes by commas ','
 scope=openid,ga4gh
```

Comment:
shouldn't this be `ga4gh_passport_v1` instead of `ga4gh` ? according to: https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md#requirement-7

Reply:
True